ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -258,8 +259,14 @@ static enum pkcs11_rc set_optional_attributes(struct obj_attrs **out,
 		void *value = NULL;
 
 		rc = get_attribute_ptr(temp, bp[n], &value, &size);
-		if (rc == PKCS11_RV_NOT_FOUND)
-			rc = get_default_value(bp[n], &value, &size);
+		if (rc == PKCS11_RV_NOT_FOUND) {
+			if (default_val)
```

Comment:
add braces since those around the `else` block.

Reply:
Done